models/ACC/ACC.jl

```diff
@@ -0,0 +1,29 @@
+using ReachabilityAnalysis
+
+@taylorize function ACC!(dx, x, p, t)
+    local u, a_lead, a_ego = 0.0001, -2., -1.
+    # x[1] = x_lead
```

Comment:
note that you can unpack the variables like so

```
x_lead, v_lead, v_lead, x_ego, v_ego, γ_ego = x
```
and then use these when writing down the rhs below. in fact i think it's clearer that way.